src/nvim/edit.c

```diff
@@ -462,6 +462,8 @@ static void insert_enter(InsertState *s)
     o_lnum = curwin->w_cursor.lnum;
   }
 
+  foldUpdateAll(curwin);
+  foldOpenCursor();
```

Comment:
What was calling `foldUpdateAll()` before now? We are adding new calls, but can we delete the old one?